lib/signs_ui/config/writer.ex

```diff
@@ -64,7 +64,7 @@ defmodule SignsUi.Config.Writer do
           if Enum.any?(ids, fn id ->
                case signs[id] do
                  nil -> false
-                 sign -> sign.config.mode == :headway
+                 sign -> sign.config.mode == :headway or sign.config.mode == :static_text
```

Comment:
nit: this could more-concisely/less-duplicatively be `sign.config.mode in [:headway, :static_text]`

Reply:
Thanks!